Makefile

```diff
@@ -119,6 +119,22 @@ hosts-vivaldi-user:
 	*)                     echo "The operating system $(OS) is not supported"; exit 1 ;; \
 	esac
 
+.PHONY: hosts-brave
+hosts-brave:
+	@case $(OS) in \
+	Linux)  			   ln -sf "$(DESTDIR)/usr/lib/browserpass/hosts/chromium/$(APP_ID).json" "/etc/opt/brave/native-messaging-hosts/" ;; \
```

Comment:
I doubt this is correct path, but `/etc/opt/chrome/native-messaging-hosts/` in https://github.com/browserpass/browserpass/pull/295/ was probably wrong too — let's see if someone confirms or fixes this :)